modAionImpl/src/org/aion/zero/impl/sync/TaskShowStatus.java

```diff
@@ -74,23 +94,40 @@ public void run() {
             String selfTd = selfBest.getCumulativeDifficulty().toString(10);
 
             String status =
-                "sync-status avg-import=" + String.format("%.2f", this.statics.getAvgBlocksPerSec())
-                    //
-                    + " b/s" //
-                    + " td=" + selfTd + "/" + networkStatus.getTargetTotalDiff().toString(10) //
-                    + " b-num=" + selfBest.getNumber() + "/" + this.networkStatus
-                    .getTargetBestBlockNumber() //
-                    + " b-hash=" + Hex.toHexString(this.chain.getBestBlockHash()) //
-                    + "/" + this.networkStatus.getTargetBestBlockHash() + "";
+                    "sync-status avg-import="
+                            + String.format("%.2f", this.statics.getAvgBlocksPerSec())
+                            //
+                            + " b/s" //
+                            + " td="
+                            + selfTd
+                            + "/"
+                            + networkStatus.getTargetTotalDiff().toString(10) //
+                            + " b-num="
+                            + selfBest.getNumber()
+                            + "/"
+                            + this.networkStatus.getTargetBestBlockNumber() //
+                            + " b-hash="
+                            + Hex.toHexString(this.chain.getBestBlockHash()) //
+                            + "/"
+                            + this.networkStatus.getTargetBestBlockHash()
+                            + "";
 
             p2pLOG.info(status);
 
+            if (p2pLOG.isDebugEnabled()) {
```

Comment:
Should it be a TRACE level? Cause this information can be huge.

Reply:
It's similar to the p2p printout which is placed in debug. I don't think it belongs in trace logging, because it's infrequent. But if you have a strong preference for moving it to trace, I will make the change.